clodius/multivec.py

```diff
@@ -20,9 +20,11 @@ def bedfile_to_multivec(input_filenames, f_out,
     '''
 
     files = []
+    in_bytes = False
     for input_filename in input_filenames:
         if op.splitext(input_filename)[1] == '.gz':
             files += [gzip.open(input_filename, 'r')]
+            in_bytes = True
```

Comment:
You can get rid of `in_bytes` and use `gzip.open(input_filename, 'rt')` to open the file as text rather than bytes.